### PR TITLE
Minor logger improvements

### PR DESCRIPTION
**Pull request by Jalle19**
@ksooo this fixes the latest Coverity complaint (at least I hope so).

### CODE REVIEW

File: src/tvheadend/utilities/Logger.h [hunk 1]
```diff
@@ -70,49 +64,24 @@ namespace tvheadend
...
-        va_end(arguments);
-
-        logger.m_implementation(level, buffer);
-      }
+      static void Log(LogLevel level, const std::string &message, ...);
 
       /**
        * Configures the logger to use the specified implementation
-       * @þaram implementation lambda
+       * @param implementation lambda
```
Comment (ksooo) [inline src/tvheadend/utilities/Logger.h]: hehe, a nice one...

**Comment by ksooo** [comment]
Looks sane. Maybe you should squash this into one commt befor merge? Not sure, though.

**Comment by Jalle19** [comment]
@ksooo I left them as separate commits since IMO they're fairly unrelated.

**Comment by Jalle19** [comment]
BTW I even build tested on Windows this time ;-)

**Comment by ksooo** [comment]
A good guy you are. ;-)

**Comment by Jalle19** [comment]
@ksooo squashed some of the commits, good to go?

**Comment by ksooo** [comment]
Shove it in.

**Pull request merged**